Polyhedron/demo/Polyhedron/CMakeLists.txt

```diff
@@ -62,6 +62,20 @@ if (EIGEN3_FOUND)
   include( ${EIGEN3_USE_FILE} )
 endif(EIGEN3_FOUND)
 
+# Activate concurrency?
+option(ACTIVATE_CONCURRENCY
+  "Enable concurrency"
+  ON)
+
+if( ACTIVATE_CONCURRENCY )
+  find_package( TBB )
+  if( TBB_FOUND )
+    include( ${TBB_USE_FILE} )
+    list( APPEND CGAL_3RD_PARTY_LIBRARIES ${TBB_LIBRARIES} )
+  else()
+    message( STATUS "NOTICE: Intel TBB NOT found! The example is faster if TBB is linked." )
+  endif()
+endif()
```

Comment:
The name `ACTIVATE_CONCURRENCY` is too generic. If one compiles CGAL-4.7 with the CMake option `WITH_demos=TRUE`, that name `ACTIVATE_CONCURRENCY` and the description "Enable concurrency" will make the user think that it activates a sort of concurrency support for the whole library.

What is more, if the Point Set processing package follows [the guidelines "Concurrency in CGAL"](https://cgal.geometryfactory.com/CGAL/Members/wiki/Guidelines/Concurrency_in_CGAL), then `include( ${TBB_USE_FILE} )` should not be sufficient to actually activate the concurrency! The file `UseTBB.cmake` (in `Installation/cmake/modules/`) only defines the macro `CGAL_LINKED_WITH_TBB`. Then the code should also use `Parallel_tag` instead of `Sequential_tag`. Or maybe that means that `Parallell_tag` is the default when TBB is available? Is that documented?